server/models/games-models.js

```diff
@@ -68,3 +68,32 @@ exports.selectReviews = () => {
       return err;
     });
 };
+exports.selectCommentsByReviewId = async (review_id) => {
+  if (isNaN(+review_id)) {
+    return Promise.reject({
+      status: 400,
+      msg: "Invalid input"
+    });
+  }
```

Comment:
This is fine and it works! But PSQL would throw an error for you too if the review_id is not a number. If you fancy refactoring feel free and see if you can catch the error in your error handler by seeing what code you get, but I'll leave that upto you!